modules/lib/lib_gui.py

```diff
@@ -1,7 +1,13 @@
 import tkinter as tk
-import pyperclip
 from tkinter import messagebox
 from lib_pass_gen import PasswordGenerator
+from utils import copy_to_clipboard
+
+
+def is_number(user_input):
+    data = user_input.get()
+    return data.isdigit() and int(data) in range(1, 257)
```

Comment:
Function is called `is_number`, we should either not check the correctness of such number here or change the function name